ORM/Connect/MySQLSchemaManager.php

```diff
@@ -454,8 +459,8 @@ public function set($values) {
 		//DB::requireField($this->tableName, $this->name, "enum('" . implode("','", $this->enum) . "') character set
 		//utf8 collate utf8_general_ci default '{$this->default}'");
 		$valuesString = implode(",", Convert::raw2sql($values['enums'], true));
-		$charset = Config::inst()->get('MySQLDatabase', 'charset');
-		$collation = Config::inst()->get('MySQLDatabase', 'collation');
+		$charset = Config::inst()->get('SilverStripe\ORM\Connect\MySQLDatabase', 'charset');
+		$collation = Config::inst()->get('SilverStripe\ORM\Connect\MySQLDatabase', 'collation');
 		return "set($valuesString) character set {$charset} collate {$collation}" . $this->defaultClause($values);
```

Comment:
We can't use ClassName::class unless we bump up the minimum php version to 5.6.

We can't use short name in config because then we'd run into ambiguous class name problems, which is the whole point of using namespaces in the first place.

A better solution is use the `Configurable` trait, and then you can just do ClassName::config()->my_config to access it.